### ISSUE TITLE
R 3.5.0

### ISSUE THREAD

**Issue by kalefranz** [ContinuumIO/anaconda-issues#9529]
_From @druchnewitz on May 25, 2018 10:49_

Hi together,

Bioconductor has updated there Package and uses R version 3.5.0.
When will it be available via Anaconda?

Best

_Copied from original issue: conda/conda#7339_

**Comment by LuckyMD** [comment]
Hi,

I'm wondering the same. Is R 3.5.0 in the pipeline?

Best,

**Comment by msarahan** [comment]
Yes, it is in the pipeline. We will release it when the corresponding MRO
version is released.

**Comment by msarahan** [comment]
Then we are working on it, and we'll have it available as soon as we can.

**Comment by hansenaw** [comment]
Hi, I'm also interested in this. Trying to install Bioconductor packages dependent on R 3.5+.

Any updates?

**Comment by almartin82** [comment]
Also interested in this.  Thanks!

**Comment by qinzhu** [comment]
Also interested in this. Really need R3.5 for bioconductor to work for my project. Thanks!

**Comment by mingwandroid** [comment]
For linux-32 and linux-64 and macOS, you can install test R 3.5.0 and for Linux you can test MRO 3.5.0 by using the `rdonnellyr` channel. Feedback very welcome!

**Comment by mopshell** [comment]
how about windows, do we have R 3.5 version ready?

**Comment by mingwandroid** [comment]
Not yet.

**Comment by Savyaschi** [comment]
For windows R 3.5 is available......

**Comment by qinzhu** [comment]
@Savyaschi Thanks for notifying!
